samples/addmoddel.cpp

```diff
@@ -2,11 +2,10 @@
 // addmoddel.cpp
 // Sample program showing how to add, modify and delete Exif metadata.
 
+#include <cassert>
 #include <exiv2/exiv2.hpp>
```

Comment:
We want to keep `#include <exiv2/exiv2.hpp>` always on top. We would need to add an additional blank line to restrict the sorting